libsolidity/parsing/Parser.cpp

```diff
@@ -104,8 +104,8 @@ ASTPointer<SourceUnit> Parser::parse(shared_ptr<Scanner> const& _scanner)
 	}
 	catch (FatalError const&)
```

Comment:
This is the code that could be put in a macro. However better would be figure out a way to put the entire try/catch into a macro. The problem is that there are variables that are often declared before the `try` and after the last `catch` and escaping each line inside the body of the `try` seems ugly. What do you think?

Reply:
#6681 puts some of this in a C preprocessor macro.